### PR TITLE
fix(nextjs): Don't add '.env.sentry-build-plugin' to .gitignore if it's already there

### PR DESCRIPTION
**Pull request by HaraldNordgren**

### CODE REVIEW

File: src/utils/clack-utils.ts [hunk 1]
```diff
@@ -614,7 +616,16 @@ SENTRY_AUTH_TOKEN=${authToken}
 }
 
 async function addCliConfigFileToGitIgnore(filename: string): Promise<void> {
-  //TODO: Add a check to see if the file is already ignored in .gitignore
+  const regex = new RegExp(`^\\s*${filename}\\s*=`, 'g');
```
Comment (Lms24) [inline src/utils/clack-utils.ts:619]: Two questions:
1. why do we need the final `=`?
2. should this have `gm` for flags? 

Couldn't get the regex to match on a simple input without changing 1 and 2 (https://regex101.com/r/0I5uLE/1).

**Review by Lms24 — COMMENTED** [review]
Hey @HaraldNordgren thanks for opening this PR! The intention looks good to me (this has been a low-prio long standing todo in my mind anyway). Just had a question about the regex.

File: src/utils/clack-utils.ts (hunk 1 above)
Comment (HaraldNordgren) [reply src/utils/clack-utils.ts]: Thanks for the help @Lms24, much appreciated! I didn’t test this regex before committing and that is definitely my bad!

It was copied wholesale from the regex for SENTRY_AUTH_TOKEN in src/utils/clack-utils.ts

Comment (Lms24) [reply src/utils/clack-utils.ts]: No worries! The auth token itself isn't added to `.gitignore` but to an env file, hence the `=` is in there. No rush, feel free to update whenever you have time :)

Comment (HaraldNordgren) [reply src/utils/clack-utils.ts]: @Lms24 Updated now! 😅

**Review by lforst — APPROVED** [review]
Thanks!

**Comment by HaraldNordgren** [comment]
@lforst Thank you!

This wizard is awesome, keep up the great work 🙌

**Pull request merged**